README.md

```diff
@@ -88,6 +88,28 @@ cat spec/fixtures/bb112zx3193.json | janeway "$.description.contributor[?@.role[
 ]
 ```
 
+### Formatting for display
+
+Methods ending in `_display_data` usually return arrays of a class called `DisplayData`, which is designed for rendering into HTML by a consuming application. Each `DisplayData` object has a `label`, which serves as a heading under which its data is grouped. The `values` method returns an array of strings, which are the individual values to be displayed under that heading.
+
+For example, when displaying contributors, the label is usually determined by the role of the contributor, and the values are the display names of the contributors with that role:
+
+```ruby
+> record.contributor_display_data.map { |dd| [dd.label, dd.values] }
+=> [["Former owner", ["Hearst Magazines, Inc."]], ["Photographer", ["Chesebrough, Jerry"]]]
+```
+
+The `#to_hash` helper method, which collapses all provided `DisplayData` into a single hash, can be used as a quick way to check the overall structure:
+
+```ruby
+> CocinaDisplay::DisplayData.to_hash(record.subject_display_data)
+=> {"Marque"=>["Bugatti"], "Model"=>["Bugatti T51A"], "Subject"=>["Bugatti automobile"]}
+```
+
+Note that usage of the `displayLabel` attribute in Cocina overrides the label that would ordinarily be used to group an item. It's possible for use of `displayLabel` to result in items being grouped together after relabeling, or to be split out into their own group.
+
+In some cases, you may need more control over the formatting. The `objects` method gives access to the underlying objects that were grouped under a particular `label`, allowing you to format them as needed. For contributors, the underlying objects are `Contributor` instances, which provide access to the associated `Name` and `Role` objects.
```

Comment:
I think it might also be helpful to show you can create your own DisplayData for custom fields. This allows for very easy formatting in applications because everything is in the same structure. See https://github.com/sul-dlss/purl/blob/f45d3757a09125a05c92019d974d5442bca32124/app/components/description_component.rb#L41-L42

Reply:
This pointed me to https://github.com/sul-dlss/cocina_display/pull/191, which will make the explanation clearer.